CMSgov/design-system
# [WNMGDS-854] Fix IE 11 'forEach' error

## Summary
This PR fixes the issue of CMSDS document site throwing `forEach` error when viewing the website with DevTools on IE 11.

*To Reproduce*
Steps to reproduce the behavior:
1. Go to https://design.cms.gov/
2. Click on any component
3. See error

## How to test
- Go to the demo doc site: http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-854/fix-ie11-foreach-error/
- View with DevTools and check that the document site works on IE11 and other browsers
